Utility/PiecesTemplates.FCMacro

```diff
@@ -0,0 +1,421 @@
+# -*- coding: utf-8 -*-
+# (c) 18Turbo, 2022
+
+__Name__ = 'PiecesTemplates'
+__Comment__ = 'Create a structured pieces (PiecesTemplates)\nCreador de estructuras de piezas'
```

Comment:
Please add a more thorough description of what the macro is doing. E.g. from an automatic of the description in Spanish.